mautic/mautic
# fixed mautic:unusedip:delete

I closed the old PR, because I had problems with the branches:
https://github.com/mautic/mautic/pull/10536

It's still the same functionality and the same code, only based on the right

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | Yes
| New feature/enhancement? (use the a.x branch)      | No
| Deprecations?                          | No
| BC breaks? (use the c.x branch)        | No
| Issue(s) addressed                     | Fixes #10387  



#### Description:
As mentioned in #10387 it was not possible to use `mautic:unusedip:delete` since Mautic 4.x. This PR fixes this Issue
#### Steps to test this PR:


1. Load up [this PR](https://mautibox.com)
2. Got to the CLI
3. execute the command `bin/console mautic:unusedip:delete`
4. There should be no exception thrown.
5. `n unused IP addresses have been deleted` should appear



<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10589"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
